src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -0,0 +1,237 @@
+
+
+## Out of the box support
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="query"
+    title="Querying your data"
+  >
+
+Metrics collected using the from Azure Monitor are:
+- Stored in New Relic as [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#dimensional-metrics) of type `summary`
+- Can be [queried](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/query-infrastructure-dimensional-metrics-nrql/) using [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/).
+- Any metrics not mapped to an entity are available as raw dimnsional metrics
+
+In order to ease migration to the Azure Monitor integration, you can continue to use the queries and metric naming from the traditional Azure integrations.
+
+All Azure Monitor metrics will have these attributes:
+
+-   `collector.name`: `azure-monitor`
+-   `instrumentation.provider`: `azure`
+-   `azure.region`
+-   `azure.resourceId`
+-   `azure.resourceType`
+-   `azure.resourceId`
+  
+  For example, to see the amount of metric updates received from Azure Monitor in the last hour, you can run this query:
+
+```
+FROM Metric SELECT count(*) WHERE collector.name = 'azure-monitor' TIMESERIES SINCE 1 hour ago
+```
+
+Custom metrics are not supported by the Azure Monitor integration at the moment.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="alerts"
+    title="Alert conditions"
+  >
+
+You can create [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/) [alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) on metrics from the New Relic Azure Monitor integration. 
+
+Make sure your filter limits data to metrics from the Azure Monitor integration only. For example:
+
+```
+FROM Metric
+SELECT max(`azure.storage.storageaccounts.SuccessServerLatency`)
+WHERE collector.name = 'azure-monitor' 
+FACET azure.region, azure.resourceId
+```
+
+Then, to make sure that alerts processes the data correctly, configure the advanced signal settings. These settings are needed because Azure Monitor receives metrics from services with a certain delay. 
+
+To configure the signal settings, under **Condition Settings**, click on **Advanced Signal Settings** and enter the following values:
+
+- **Aggregation window**: We recommend setting it to 1 minute. If you are having issues with flapping alerts or alerts not triggering, consider increasing it to 2 minutes.
+- **Offset evaluation by**: Depending on the service, CloudWatch may send metrics with a certain delay. The value is set in windows. With a 1-minute aggregation window, setting the offset to 8 ensures the majority of the metrics are evaluated correctly. You may be able to use a lower offset if the delay introduced by AWS and Firehose is less.
```

Comment:
this part is wrong it is talking about Cloudwatch and AWS and this is MicroSoft and Azure

Reply:
Correct. Actually the new Azure Monitor is not sending anything, as we pull.
Ideas on wording @faltimiras @jordigarcl ?

Reply:
I think this is a left-over from the simmilar alert guidelines we have for AWS CW MS. Maybe I'd recommend wording as just:

> With a 1-minute aggregation window, we recommend setting a 8-minute evalaution offset.